lib/rules/require-semicolons.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Requires semicolon after:
+ *
+ * * var declaration
+ * * expression statement
+ * * return
+ * * throw
+ * * break
+ * * continue
+ * * do-while
+ *
+ * Type: `Boolean`
+ *
+ * Values: `true`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireSemicolons": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * var a = 1;
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * var a = 1
+ * ```
+*/
+
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+
+    configure: function(requireSemicolons) {
+        assert(
+            typeof requireSemicolons === 'boolean',
```

Comment:
A few of the rules are checking boolean and true - we should probably remove those so no one gets confused